td/tracking/views.py

```diff
@@ -290,13 +478,15 @@ def get(self, request, *args, **kwargs):
             return redirect("tracking:project_list")
 
         allowed_urls = [
-            "http://localhost:8000/tracking/charter/new/",
-            "http://localhost:8000/tracking/event/new/",
-            "http://td.unfoldingword.org/tracking/charter/new/",
-            "http://td.unfoldingword.org/tracking/event/new/",
+            re.compile(r"^/tracking/charter/new/$"),
```

Comment:
I think you could get the string you are looking for with reverse and pass to `re.compile`:

```python
re.compile(r"^{}$".format(reverse("<name>")))
```